app/scripts/controllers/edit/deploymentConfig.js

```diff
@@ -157,39 +157,40 @@ angular.module('openshiftConsole')
       return ($scope.strategyData.type !== 'Custom' && $scope.originalStrategy !== 'Custom' && $scope.strategyData.type !== $scope.originalStrategy);
     };
 
+    var promptToMoveParams = function(pickedStrategyParams) {
+      if (!_.has($scope.strategyData, pickedStrategyParams)) {
```

Comment:
I prefer an early return rather than wrapping everything in an `if`. It's just a little easier to follow.

```js
if (_.has($scope.strategyData, pickedStrategyParams)) {
  return;
}

// Show modal...
```